rllib/common.py

```diff
@@ -0,0 +1,192 @@
+import argparse
+from dataclasses import dataclass
+from enum import Enum
+import typer
+
+from ray.tune.experiment.config_parser import _make_parser
+from ray.tune.result import DEFAULT_RESULTS_DIR
+
+
+def _create_tune_parser_help():
+    """Create a Tune dummy parser to access its 'help' docstrings."""
+    parser = _make_parser(
+        parser_creator=None,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    return parser.__dict__.get("_option_string_actions")
+
+
+PARSER_HELP = _create_tune_parser_help()
+
+
+def get_help(key: str) -> str:
+    """Get the help string from a parser for a given key.
+    If e.g. 'resource_group' is provided, we return
+    the entry for '--resource-group'."""
+    key = "--" + key
+    key = key.replace("_", "-")
+    if key not in PARSER_HELP.keys():
+        raise ValueError(f"Key {key} not found in parser.")
+    return PARSER_HELP.get(key).help
+
+
+class FrameworkEnum(str, Enum):
+    """Supported frameworks for RLlib, used for CLI argument validation."""
+
+    tf = "tf"
+    tf2 = "tf2"
+    tfe = "tfe"
```

Comment:
Let's get rid of "tfe". It's the same as "tf2" and should not be used anymore.

I know we still need to hard-deprecate it everywhere, but we shouldn't introduce it in new code anymore.

Reply:
sure, more than happy to! I find this confusing as well